proto/zitadel/policy.proto

```diff
@@ -49,24 +98,72 @@ enum PasswordlessType {
 
 message PasswordComplexityPolicy {
     zitadel.v1.ObjectDetails details = 1;
-    uint64 min_length = 2;
-    bool has_uppercase = 3;
-    bool has_lowercase = 4;
-    bool has_number = 5;
-    bool has_symbol = 6;
-    bool is_default = 7;
+    uint64 min_length = 2 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            example: "\"8\""
+        }
+    ];
+    bool has_uppercase = 3 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the password MUST contain an upper case letter"
+        }
+    ];
+    bool has_lowercase = 4 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the password MUST contain a lower case letter"
+        }
+    ];
+    bool has_number = 5 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the password MUST contain a numer"
+        }
+    ];
+    bool has_symbol = 6 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the password MUST contain a symbol. E.g. \"$\""
+        }
+    ];
+    bool is_default = 7 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the organisation's admin changed the policy"
+        }
+    ];
 }
 
 message PasswordAgePolicy {
     zitadel.v1.ObjectDetails details = 1;
-    uint64 max_age_days = 2;
-    uint64 expire_warn_days = 3;
-    bool is_default = 4;
+    uint64 max_age_days = 2 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "Maximum days since last password change"
+            example: "\"365\""
+        }
+    ];
+    uint64 expire_warn_days = 3 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "Days before the password expiry the user gets notified to change the password"
+            example: "\"10\""
+        }
+    ];
+    bool is_default = 4 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the organisation's admin changed the policy"
+        }
+    ];
 }
 
 message PasswordLockoutPolicy {
     zitadel.v1.ObjectDetails details = 1;
-    uint64 max_attempts = 2;
+    uint64 max_attempts = 2 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "Maximum attempts before the account gets locked. Attempts are reset as soon as the password is entered correct or the password is reset."
+            example: "\"10\""
+        }
+    ];
+    // TODO: how to describe?
     bool show_lockout_failure = 3;
```

Comment:
enables if the failure should be shown to de user, sometimes for security issues the user should not get to much information